index.markdown

```diff
@@ -311,32 +274,47 @@ redirect_from: '/moveit/'
                 <div class="release-versions__body">
                   <div class="release-versions__content">
                     <div class="release-versions__1">
-                      <h3>Rolling 2.4</h3>
+                      <h3>Rolling 2.5</h3>
```

Comment:
What version should we call Rolling? Here i've made it same as Humble.

Reply:
Rolling is 2.5 now, once we cut off humble support from main branch it will be 2.6 which will eventually turn into Iron.

Reply:
ok so this is correct for now, sgtm